rustls/src/client/client_conn.rs

```diff
@@ -727,6 +730,15 @@ mod connection {
             self.inner.core.data.ech_status
         }
 
+        /// Return true if the connection was made with a `ClientConfig` that is FIPS compatible.
+        ///
+        /// This is different from [`crate::crypto::CryptoProvider::fips()`]:
+        /// it is concerned only with cryptography, whereas this _also_ covers TLS-level
+        /// configuration that NIST recommends, as well as ECH HPKE suites if applicable.
+        pub fn fips(&self) -> bool {
```

Comment:
Probably warrants a tests that calls it in api.rs.

Reply:
I added a test for the `ClientConfig::fips()` update. Still TODO: `ClientConnection::fips()` coverage.

Reply:
> Still TODO: ClientConnection::fips() coverage.

Added.